apps/article/templates/article.jade

```diff
@@ -8,7 +8,15 @@ append locals
   - bodyClass = bodyClass + ' ' + article.getBodyClass()
 
 block body
-  #articles-show
+
+  //- Render channel nav if article is in a team channel
+  - var showClass = ''
+  if sd.ARTICLE_CHANNEL && sd.ARTICLE_CHANNEL.isTeam()
```

Comment:
This `.isTeam()` seems odd. `sd` should be left for the "pure" JSON data. In this case you probably want to do `articleChannel = new Channel sd.ARTICLE_CHANNEL` first.